leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -143,6 +145,29 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
     disableDumpHeapInTests()
   }
 
+  private fun checkAllowInNonDebuggableBuild() {
```

Comment:
nit: maybe rename to `checkRunningInDebuggableBuild()` ? We need a method name that will be clear to consumers when they get this error in their build